.github/workflows/mvn-verify-check.yml

```diff
@@ -24,7 +24,61 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  mvn-verify-check:
+  get-noSnapshot-versions-from-dist:
+    runs-on: ubuntu-latest
+    outputs:
+      sparkHeadVersion: ${{ steps.noSnapshotVersionsStep.outputs.headVersion }}
```

Comment:
looks like we are using this headVersion(311) here? for `verify-all-modules-with-headSparkVersion` we use default buildver

Reply:
correct, I ended not needing it. But actually it makes the flow more clear if I use it explicitly. I'll update